.github/workflows/molecule.yml

```diff
@@ -10,33 +10,33 @@ on:
 jobs:
 
   list-scenarios:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     outputs:
       matrix: ${{ steps.listscenarios.outputs.scenarios }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - id: listscenarios
         uses: ome/action-ansible-molecule-list-scenarios@main
 
   test:
     name: Test
     needs:
       - list-scenarios
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     strategy:
       # Keep running so we can see if other tests pass
       fail-fast: false
       matrix:
         scenario: ${{fromJson(needs.list-scenarios.outputs.matrix)}}
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v4
+      - uses: actions/setup-python@v5
         with:
-          python-version: '3.9'
+          python-version: '3.12'
       - name: Install Ansible & Molecule
         run: |
-            pip install "ansible<8" "ansible-lint<6.13" flake8
-            pip install "molecule<5" "ansible-compat<4"
+            pip install "requests" "ansible < 12" "ansible-lint" "flake8"
```

Comment:
The Ansible package version cap should be removed, so the role can now be tested against the latest available Ansible release (including Ansible 12 once it becomes available).

Reply:
fixed in https://github.com/ome/ansible-role-omero-web/pull/52/commits/4eb9952330c1d10570a675a0310fdd79cc761391